docs/examples/201/vmss-windows-autoscale/main.bicep

```diff
@@ -0,0 +1,232 @@
+param vmSku string = 'Standard_A1_v2'
+param windowsOSVersion string {
+  default: '2019-Datacenter'
+  allowed: [
+    '2019-Datacenter'
+    '2016-Datacenter'
+    '2012-R2-Datacenter'
+    '2012-Datacenter'
+  ]
+}
+param vmssName string {
+  maxLength: 61
+}
+param instanceCount int {
+  maxValue: 100
+  minValue: 1
+}
+param adminUsername string
+param adminPassword string {
+  secure: true
+}
+param location string = resourceGroup().location
+
+var namingInfix = toLower(substring(concat(vmssName, uniqueString(resourceGroup().id)), 0, 9))
```

Comment:
use string interpolation instead of `concat()` and use `vmss.name` instead of `vmssName`

Reply:
I can't use vss.name because of a reference cycle.
 
![Capture d’écran 2020-11-30 à 17 17 48](https://user-images.githubusercontent.com/5141735/100634915-28bf3b00-3330-11eb-805c-d8ecec9081a5.png)